src/features/myProtectedData/newProtectedData/NewProtectedData.tsx

```diff
@@ -111,7 +124,8 @@ export default function NewProtectedData() {
 
       <h2>Protect New Data</h2>
       <p className="-mt-3 mb-4">
-        Protect new email or file: encrypt, monetize and control access.
+        Protect new email, telegram or file : encrypt, monetize and control
```

Comment:
```suggestion
        Protect an email address, a telegram chat ID or a file: encrypt, monetize and control
```

Petite note : il n'y a pas d'espace avant les deux points (`:`) en Anglais.